cluster/ci/config/roles.yaml

```diff
@@ -868,3 +868,52 @@ items:
     kind: ClusterRole
     apiGroup: rbac.authorization.k8s.io
     name: system:image-puller
+
+- kind: Namespace
+  apiVersion: v1
+  metadata:
+    name: open-cluster-management
+    annotations:
+      purpose: "RedHat Advanced Cluster Management Build Artifacts"
+- kind: Group
+  apiVersion: v1
+  metadata:
+    name: ocm-admins
+  users:
+  - ray-harris
+  - arewm
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: admin
+    namespace: open-cluster-management
+  subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: ocm-admins
+  - kind: Group
+    name: ci-admins
+    apiGroup: rbac.authorization.k8s.io
+  - kind: ServiceAccount
+    name: ci-operator
```

Comment:
Why you need this?

Reply:
When this job ran: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/logs/branch-ci-open-cluster-management-image-builder-master-images/1

It failed with this error:
```error: could not run post step : could not ensure target imagestream: imagestreams.image.openshift.io "builder" is forbidden: User "system:serviceaccount:ci:ci-operator" cannot get imagestreams.image.openshift.io in the namespace "open-cluster-management": no RBAC policy matched```

After running this command:
```oc adm policy add-role-to-user admin system:serviceaccount:ci:ci-operator -n open-cluster-management```

The job run successfully and pushed the image. There may be more restrictive permissions that will work or some other way to get around the error, but that's what worked.

Reply:
You do not need to be project admin to push.
See `image-tagger-ci-operator` for example.

Reply:
Ok. Pushing commit to fix.